feat: ui.table context menu items (#522)
deephaven/deephaven-plugins: plugins/ui/docs/README.md

![Table events](table_events.png)

### ui.table Context Menu

> Items can be added to the bottom of the `ui.table` context menu (right-click menu) by using the `context_menu` or `context_header_menu` props. The `context_menu` prop adds items for each cell, while the `context_header_menu` prop adds items for the header cells.

Reviewer: I know we don't show the Row Headers right now, question is if we'd ever want to allow that on `ui.table` and if so, what would the context menu item be for that? `context_row_header_menu` I think would be acceptable, and `context_header_menu` is for column headers.

In any case, I think we should be explicit this is about column headers in the description here.

```suggestion
Items can be added to the bottom of the `ui.table` context menu (right-click menu) by using the `context_menu` or `context_header_menu` props. The `context_menu` prop adds items to the cell context menu, while the `context_header_menu` prop adds items to the column header context menu.
```
